customer.rb

```diff
@@ -1 +1,44 @@
-# See README.md
+class Customer
+  @@all = []
+  attr_accessor :id, :first_name, :last_name, :location, :orders, :money_spent, :amount_of_orders
```

Comment:
Make sure to only use `attr_accessor`, with you need to modify the attributes from outside the object. I think this isn‘t the case here, so better use `attr_reader`

```suggestion
  attr_reader :id, :first_name, :last_name, :location, :orders, :money_spent, :amount_of_orders
```